hackclub.com.yaml

```diff
@@ -319,7 +319,7 @@ adastra:
 astra:
   - ttl: 600
     type: CNAME
-    value: astra-collective.github.io.
+    value: astra2-ops.github.io.
```

Comment:
What is this change?

Reply:
![image](https://github.com/user-attachments/assets/c5a77a7b-5f8f-40ff-a6ed-d037ce2c2b8d)
Hi the chang in the the above img was as i shifted the the github user 
![image](https://github.com/user-attachments/assets/64a043e3-ede0-43e7-82d0-17800d7c90d2)
this above shown change is me (from @AVSCGREAT ) adding my to be personal website

Reply:
I'm still confused on what this subdomain is for? Is it an official HQ project or a club?

Reply:
@AVSCGREAT @astra2-ops ^